assets/css/project/info.less

```diff
@@ -25,7 +31,6 @@
 		}
     }
 
-body p.source_info, /* cascade for weight */
```

Comment:
How did you get those changes in your branch? It was my refactoring from other branch.